Templates/Tests/Expected/AutoMockable.expected

```diff
@@ -1459,7 +1484,7 @@ class StaticMethodProtocolMock: StaticMethodProtocol {
 
     static func staticFunction(_ : String) -> String {
         staticFunctionStringStringCallsCount += 1
-        staticFunctionStringStringReceived = 
```

Comment:
This change needs to be reverted, because the equality check would fail during tests.

Problem occurs if to open this file in Xcode, which truncates trailing whitespace.

Reply:
Fixed